spp_import_dci_api/models/group_membership.py

```diff
@@ -0,0 +1,16 @@
+from odoo import api, fields, models
+
+
+class OpenSPPGroupMembership(models.Model):
+    _inherit = "g2p.group.membership"
+
+    is_created_from_crvs = fields.Boolean(
+        "Created from CRVS",
+        compute="_compute_is_created_from_crvs",
+        store=True,
+    )
+
+    @api.depends("group")
+    def _compute_is_created_from_crvs(self):
+        for rec in self:
+            rec.is_created_from_crvs = rec.group.grp_is_created_from_crvs
```

Comment:
```suggestion
    is_created_from_crvs = fields.Boolean(
        "Created from CRVS",
        related="group.grp_is_created_from_crvs",
        store=True,
    )
```
This will be shorter